src/screens/MainStack/Tutorial/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {FC} from 'react';
+import {Image} from 'react-native';
+
+import Swiper from 'react-native-swiper';
+import styled from 'styled-components/native';
+
+import {TUTORIAL_PAGE_1, TUTORIAL_PAGE_2, TUTORIAL_PAGE_3} from '~/assets';
+import BottomModal from '~/components/BottomModal';
+import {ContainerStyle} from '~/utils/Styles';
+
+const Container = styled.View`
+  ${ContainerStyle}
+`;
+
+const Tutorial: FC = () => {
+  return (
+    <Swiper>
+      <Container>
```

Comment:
❓ QUESTION
Doesn't `SafeAreaView` need? I think it's necessary when you add bottom button at last image.